moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp

```diff
@@ -134,6 +139,10 @@ ompl_interface::PlanningContextManager::plannerSelector(const std::string& plann
 
 void ompl_interface::PlanningContextManager::registerDefaultPlanners()
 {
+  registerPlannerAllocator(                //
```

Comment:
whats with the // ?

Reply:
I don't know, I'm pretty sure they don't have a meaningful function ;) I just copied the pattern from the lines.